api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -267,6 +267,9 @@ public static void setContext(ServiceContext ctx) {
 	}
 
 	/**
+	 * @deprecated
```

Comment:
Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Deprecation

Reply:
@dkayiwa I did this in my last commit:
```java
/**
 * @deprecated as of 2.3.0, replaced by #authenticate(AuthenticationScheme, Credentials)
 ...
 */
```
Is that what you expect? If not please point me to the exact JavaDoc that you would like to have in place.